probing/gap_preprocess.py

```diff
@@ -0,0 +1,137 @@
+import retokenize_edge_data as retokenize
+import sys
+import argparse
+import pandas as pd
+import pickle
+from enum import Enum
+
```

Comment:
Add a brief introduction on what GAP is and what extra preprocessing you need for it.

Reply:
(And generally add a few general comments below to explain the structure of what's happening here.)